Core/src/Geometry/CylinderVolumeHelper.cpp

```diff
@@ -499,7 +499,7 @@ bool Acts::CylinderVolumeHelper::estimateAndCheckDimension(
   double zEstFromLayerEnv = 0.5 * ((layerZmax) + (layerZmin));
   double halflengthFromLayer = 0.5 * std::abs((layerZmax) - (layerZmin));
 
-  bool concentric = (zEstFromLayerEnv * zEstFromLayerEnv < 0.001);
+  bool concentric = std::abs(zEstFromLayerEnv) < 0.031622777;
```

Comment:
Where's this number from?

Reply:
this is `sqrt(0.001)`. I could change it to that actually. what do you think?